kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -487,6 +496,32 @@ def _submit_container_state_metric(self, metric_name, state_name, c_state, state
             gauge_name = '{}.containers.{}.{}'.format(self.NAMESPACE, metric_name, state_name)
             self.gauge(gauge_name, 1, tags + reason_tags)
 
+    def _report_ephemeral_storage_usage(self, pod_list, instance_tags):
+        stats = self._retrieve_stats()
+
+        ephemeral_storage_usage = {}
+        for pod in stats['pods']:
```

Comment:
```suggestion
        for pod in stats.get('pods'):
```
to avoid a `KeyError` exception

Reply:
You'll get a `TypeError` though.

Reply:
It even needs to be `for pod in stats.get('pods', []):` to avoid the `unable to iterate on None` error.